ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/vm/DebugOperationTracer.java

```diff
@@ -68,7 +69,12 @@ public void traceExecution(
               stack,
               memory,
               storage,
-              frame.getRevertReason().orElse(null)));
+              frame
+                  .getRevertReason()
+                  .map(
+                      revertReason ->
+                          new String(revertReason.extractArray(), StandardCharsets.UTF_8))
+                  .orElse(null)));
```

Comment:
This is still wrong, the revert reason isn't a string - we need to output the result as a hex array in the trace output.